doc/sphinx/index.latex.rst

```diff
@@ -0,0 +1,81 @@
+==========================
+ The Coq Reference Manual
+==========================
+
+Introduction
+------------
+
+.. include:: introduction.rst
+
+Credits
+-------
+
+.. include:: credits.rst
```

Comment:
Credits must not be part of the introduction, and certainly not come before the table of contents. See also #6573.

Reply:
> Credits must not be part of the introduction

They are not :)

> and certainly not come before the table of contents. See also #6573.

They don't ^^

The source of the confusion might be that in LaTeX, the table of contents is generated from the `toctree`s, but it always appears at the very beginning of the document.

Reply:
OK perfect. I couldn't check the rendered version yet so I had to guess from the code.